core/store/src/main/java/org/locationtech/geowave/core/store/callback/DuplicateCountCallback.java

```diff
@@ -0,0 +1,58 @@
+package org.locationtech.geowave.core.store.callback;
+
+import java.io.Closeable;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.locationtech.geowave.core.index.ByteArray;
+import org.locationtech.geowave.core.index.InsertionIds;
+import org.locationtech.geowave.core.index.SinglePartitionInsertionIds;
+import org.locationtech.geowave.core.store.CloseableIterator;
+import org.locationtech.geowave.core.store.adapter.InternalDataAdapter;
+import org.locationtech.geowave.core.store.api.Index;
+import org.locationtech.geowave.core.store.api.Query;
+import org.locationtech.geowave.core.store.api.QueryBuilder;
+import org.locationtech.geowave.core.store.base.BaseDataStore;
+import org.locationtech.geowave.core.store.callback.DeleteCallback;
+import org.locationtech.geowave.core.store.entities.GeoWaveRow;
+import org.locationtech.geowave.core.store.query.constraints.InsertionIdQuery;
+import org.locationtech.geowave.core.store.util.DataStoreUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
```

Comment:
This file is only used in the ITs? In which case would it make more sense to have it in the ITs even as a static private inner  class? Also, minor nit is the javadoc header looks to be a copy of the DuplicateDeletionCallback and not accurate for this class.